browser/ui/webui/settings/brave_settings_localized_strings_provider.cc

```diff
@@ -72,6 +73,9 @@ const char16_t kDebounceLearnMoreUrl[] =
 const char16_t kEnableNftDiscoveryLearnMoreUrl[] =
     u"https://github.com/brave/brave-browser/wiki/"
     u"NFT-Discovery";
+#if BUILDFLAG(ENABLE_REQUEST_OTR)
+const char16_t kRequestOTRLearnMoreUrl[] = u"https://brave.com/#TODO";
```

Comment:
TODO?

Reply:
After further discussions with Pete, we will not be needing this, so I will remove it and removing the a-href in the settings HTML